### DISCUSSION TITLE
How can i disable the auto_slice_resolution

### DISCUSSION THREAD

**Discussion by claudio9russo7** [obss/sahi#636] [Q&A] [answered]
Hi, during the preparation of a dataset I encountered a problem. I use sahi + yolov5. I sliced the images for the training of the models but just give me an error about some images truncated. As I have used sahi in the past and had no problems I have used an older version where auto_slice_resolution is not present. With the same dataset, but processed with an older version of sahi (0.9.2) I managed to train the CNN without error. I think there is a problem with auto_slice_resolution.

The error is like:
Os Error: image file is truncated (114 bytes not processed)
The environment used is kaggle

**Comment by fcakyon** [comment]
Hello @claudio9russo7, `auto_slice_resolution` doesn not affect `coco slice` command.

Can you please write down the function or script you use?

Can you also paste here the full error traceback?

**Comment by claudio9russo7** [comment]
Sure:
![coco slice](https://user-images.githubusercontent.com/99536010/187287721-0cc44b26-6b18-4aa8-bcfd-7e73fe508dc2.png)

Just make this for slicing the dataset, after this  I convert the .json file into usable format for yolo and train the network on kaggle.

**Comment by fcakyon** [comment]
Thanks @claudio9russo7!

When do you get the error? When running `slice_coco` function?

Can you also post the full error traceback?

Are you having this error with `sahi==0.9.4` or `sahi==0.10.1`?

**Comment by claudio9russo7** [comment]
no this part works perfectly, but when I go to train the network with the first dataset I had that error, sliced with the old version I had no errors and the training was successful. 
Error with the latest version
About the error traceback i need to re-slicing because just deleted everything as i managed to fix with older version.

EDIT: Slicing in this moment after preparation i wil try to train 
![coco1slice](https://user-images.githubusercontent.com/99536010/187290376-a6e869b0-fdd8-4638-b22a-dd92638d22e3.png)

**Comment by fcakyon** [comment]
Which SAHI version are you trying currently? if `sahi==0.9.4` fails, you can try with `sahi==0.9.3`. 

If `sahi==0.9.3` works and `sahi==0.9.4` fails, then this PR might be the reason: https://github.com/obss/sahi/pull/445

@devrimcavusoglu do you have any idea?

**Comment by claudio9russo7** [comment]
Im trying with `sahi==0.10.4` to re-check the problem

**Comment by fcakyon** [comment]
If you could try with `sahi==0.9.4` it would be better. Because we already know `sahi==0.10.4` has a problem.

**Comment by claudio9russo7** [comment]
As you requested I tried with `sahi==0.9.4` and no error for now during the train (last time the error appeared at the first epoch), so maybe some images get corrupted durin upload/download i dont'know

**Comment by fcakyon** [comment]
It would be great if this is only a file corruption during upload/download.

Then can you try everything again with `sahi==0.10.4` and see if the first epoch finishes without any error?

**Comment by claudio9russo7** [comment]
Yep, no problem but i will try tomorrow.

**Comment by claudio9russo7** [comment, marked as answer]
Okkei i just used `sahi==0.10.4`, run the training and no problem. So, i think was only an error during the upload on wandb of the dataset.
